[16][IMP] metoda noua citire lista mesaje anaf

adrian-dks opened the pull request:
Schimbare metoda la citire mesaje din SPV, implementare metoda cu paginatie.

dhongu committed:
Adaugat modul dvi

dhongu committed:
modulul depinde si de 'sale'

dhongu committed:
test actionare buton DVI din factura

dhongu committed:
adaugare readme la DVI

feketemihai committed:
[13.0] [IMP] Fix pre-commit.

OCA-git-bot committed:
l10n_ro_dvi 13.0.1.1.0

feketemihai committed:
[14.0] [MIG] Migrate l10n_ro_dvi

OCA-git-bot committed:
[UPD] README.rst

Commit:
[UPD] Update l10n_ro_dvi.pot

feketemihai committed:
14.0 Improve price difference and dvi

OCA-git-bot committed:
l10n_ro_dvi 14.0.1.2.0

stananatanaela committed:
update pre-migration

OCA-git-bot committed:
l10n_ro_dvi 14.0.1.3.0

mcojocaru committed:
[14.0][FIX] dvi to include transport from bill

OCA-git-bot committed:
l10n_ro_dvi 14.0.1.4.0

mcojocaru committed:
[FIX] l10n_ro_dvi to include transport

OCA-git-bot committed:
l10n_ro_dvi 14.0.1.5.0

feketemihai committed:
WIP: - 14.0 add l10n ro stock account fifo (#323)

* Add stock account fifo.

* 14.0 - Improve stock fifo

* Update dvi tests, stock move line completion

* rebase 14.0 + camp locatie in svl revaluation wizard

* [14.0][UPD]  fix for consume and production svl type

* [14.0][UPD]  fix tests

* [14.0][UPD] migrare si init_hook pentru svl lot_ids si locatii

* modificari pentru landed cost dupa un sale

* corectie test pentru landed cost dupa un sale

* fix migration

* fixuri pentru loturi

* svl_lot_allocation din confif sa fie vizibil doar daca stock mod este instalat

Co-authored-by: Cojocaru Marcel <marcel.cojocaru@gmail.com>

feketemihai committed:
Update translations

OCA-git-bot committed:
l10n_ro_dvi 14.0.1.6.0

mcojocaru committed:
fix izolare l10n_dvi

Commit:
[UPD] Update l10n_ro_dvi.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-14.0/l10n-romania-14.0-l10n_ro_dvi
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-14-0/l10n-romania-14-0-l10n_ro_dvi/

mcojocaru committed:
fix menu DVI

Commit:
[UPD] Update l10n_ro_dvi.pot

OCA-git-bot committed:
l10n_ro_dvi 14.0.1.7.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-14.0/l10n-romania-14.0-l10n_ro_dvi
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-14-0/l10n-romania-14-0-l10n_ro_dvi/

mcojocaru committed:
[14.0][IMP] small improvements l10n.ro.account.dvi, l10n_ro_nc_code

Commit:
[UPD] Update l10n_ro_dvi.pot

OCA-git-bot committed:
l10n_ro_dvi 14.0.1.8.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-14.0/l10n-romania-14.0-l10n_ro_dvi
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-14-0/l10n-romania-14-0-l10n_ro_dvi/

stananatanaela committed:
[IMP] l10n_ro_dvi: black, isort, prettier

Commit:
[UPD] Update l10n_ro_dvi.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
l10n_ro_dvi 15.0.1.1.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-15.0/l10n-romania-15.0-l10n_ro_dvi
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-15-0/l10n-romania-15-0-l10n_ro_dvi/

dhongu committed:
Translated using Weblate (Romanian)

Currently translated at 64.2% (70 of 109 strings)

Translation: l10n-romania-15.0/l10n-romania-15.0-l10n_ro_dvi
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-15-0/l10n-romania-15-0-l10n_ro_dvi/ro/

OCA-git-bot committed:
[UPD] README.rst

stananatanaela committed:
[15.0][IMP] diferenta de pret in dvi

Commit:
[UPD] Update l10n_ro_dvi.pot

OCA-git-bot committed:
oca-github-bot post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-15.0/l10n-romania-15.0-l10n_ro_dvi
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-15-0/l10n-romania-15-0-l10n_ro_dvi/

dhongu committed:
Corectie DVI

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #645 into 16.0

Signed-off-by dhongu

Commit:
[UPD] Update l10n_ro_account_period_close.pot

OCA-git-bot committed:
[BOT] post-merge updates

dhongu committed:
adaugare script de migrare

Commit:
[UPD] Update l10n_ro_dvi.pot

OCA-git-bot committed:
[BOT] post-merge updates

stananatanaela committed:
[16.0][UP] update invoice_report

OCA-git-bot committed:
Merge PR #672 into 16.0

Signed-off-by feketemihai

OCA-git-bot committed:
[BOT] post-merge updates

dhongu committed:
mig

dhongu committed:
l10n_ro_property_customs_commision_product_id

dhongu committed:
fix can't write on invisible field vat_price_difference_product_id

OCA-git-bot committed:
[IMP] update dotfiles

Commit:
[UPD] Update l10n_ro_account_anaf_sync.pot

dhongu committed:
confgurare limite plata cash

OCA-git-bot committed:
Merge PR #686 into 16.0

Signed-off-by dhongu

Commit:
[UPD] Update l10n_ro_account.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_account
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_account/

vasi26ro committed:
Added tests

dhongu committed:
Merge branch '16.0' into 16.0-mig-l10n_ro_dvi

OCA-git-bot committed:
Merge PR #659 into 16.0

Signed-off-by dhongu

Commit:
[UPD] Update currency_rate_update_RO_BNR.pot

OCA-git-bot committed:
[BOT] post-merge updates

dhongu committed:
determinare judet in functie de codul postal

dhongu committed:
verificare cui la salvare

dhongu committed:
update

dhongu committed:
Merge branch '16.0-city' into 16.0-partner_by_vat

dhongu committed:
merge city

dhongu committed:
eliminare fisiere traducere

dhongu committed:
eFactura

dhongu committed:
fix test

OCA-git-bot committed:
Merge PR #695 into 16.0

Signed-off-by dhongu

Commit:
[UPD] Update l10n_ro_city.pot

Commit:
[UPD] Update l10n_ro_partner_create_by_vat.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_partner_create_by_vat
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_partner_create_by_vat/

dhongu committed:
date

dhongu committed:
fix

dhongu committed:
fix tests

OCA-git-bot committed:
Merge PR #699 into 16.0

Signed-off-by dhongu

Commit:
[UPD] Update l10n_ro_account_anaf_sync.pot

Commit:
[UPD] Update l10n_ro_account_edi_ubl.pot

OCA-git-bot committed:
[BOT] post-merge updates

dhongu committed:
adaugare teste

OCA-git-bot committed:
Merge PR #704 into 16.0

Signed-off-by dhongu

Commit:
[UPD] Update l10n_ro_account_edi_ubl.pot

OCA-git-bot committed:
[BOT] post-merge updates

stananatanaela committed:
[16.0] adaugare camp payment disposal in view si modificare _post pe account_move sa creeze statement

dhongu committed:
Merge branch '16.0' into 16.0-mig-l10n_ro_dvi

stananatanaela committed:
update manifest.py

stananatanaela committed:
scoatere xml

stananatanaela committed:
update test_bank_statement_line_name

stananatanaela committed:
[16.0] rulare metoda check_vat_on_payment la create si write pe partener

OCA-git-bot committed:
Merge PR #719 into 16.0

Signed-off-by feketemihai

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #717 into 16.0

Signed-off-by feketemihai

OCA-git-bot committed:
[BOT] post-merge updates

dhongu committed:
update

dhongu committed:
modificari din 15

dhongu committed:
precommit

dhongu committed:
efactura
anaf-sync-multicompany

dhongu committed:
adaugare fisier istoric.txt

stananatanaela committed:
[16.0] update _l10n_ro_get_price_difference_check_action

OCA-git-bot committed:
Merge PR #724 into 16.0

Signed-off-by feketemihai

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #723 into 16.0

Signed-off-by feketemihai

Commit:
[UPD] Update l10n_ro_account_anaf_sync.pot

Commit:
[UPD] Update l10n_ro_account_edi_ubl.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_account_edi_ubl
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_account_edi_ubl/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_account_anaf_sync
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_account_anaf_sync/

dhongu committed:
Merge branch '16.0' into 16.0-mig-l10n_ro_dvi

dhongu committed:
fix

dhongu committed:
eliminare tesete esuate

dhongu committed:
fix test script

OCA-git-bot committed:
Merge PR #728 into 16.0

Signed-off-by dhongu

OCA-git-bot committed:
[BOT] post-merge updates

dhongu committed:
Merge branch '16.0' into 16.0-mig-l10n_ro_dvi

OCA-git-bot committed:
Merge PR #675 into 16.0

Signed-off-by dhongu

Commit:
[UPD] Update l10n_ro_config.pot

Commit:
[UPD] Update l10n_ro_dvi.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_config
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_config/

stananatanaela committed:
[16.0] update l10n_ro_dvi

stananatanaela committed:
update test

OCA-git-bot committed:
Merge PR #733 into 16.0

Signed-off-by feketemihai

OCA-git-bot committed:
[BOT] post-merge updates

feketemihai committed:
[FIX] [15.0] Fix price base quantity

feketemihai committed:
Update res_partner.py

OCA-git-bot committed:
Merge PR #738 into 16.0

Signed-off-by feketemihai

OCA-git-bot committed:
[BOT] post-merge updates

feketemihai committed:
[16.0] [IMP] Remove manual config, improve error handling in step2.

feketemihai committed:
[16.0] [IMP] Allow downloading e-invoices from ANAF, Add pdf embedded when the xml file is generate

feketemihai committed:
Fix message body.

dhongu committed:
sleep_time

dhongu committed:
adaugare fisier istoric.txt

OCA-git-bot committed:
Merge PR #754 into 16.0

Signed-off-by dhongu

OCA-git-bot committed:
[BOT] post-merge updates

dhongu committed:
_get_move_applicability

dhongu committed:
no anaf_config

feketemihai committed:
Update city.

OCA-git-bot committed:
Merge PR #748 into 16.0

Signed-off-by feketemihai

feketemihai committed:
Update account_edi_xml_cius_ro.py

Commit:
[UPD] Update l10n_ro_account_edi_ubl.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_account_anaf_sync
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_account_anaf_sync/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_account_edi_ubl
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_account_edi_ubl/

OCA-git-bot committed:
Merge PR #740 into 16.0

Signed-off-by feketemihai

OCA-git-bot committed:
[BOT] post-merge updates

dhongu committed:
Add neutralize.sql for test state setup

dhongu committed:
fix precommit

dhongu committed:
Update URL in neutralize.sql for l10n_ro_account_anaf_sync

OCA-git-bot committed:
Merge PR #757 into 16.0

Signed-off-by dhongu

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #745 into 16.0

Signed-off-by dhongu

Commit:
[UPD] Update l10n_ro_account_edi_ubl.pot

Commit:
[UPD] Update l10n_ro_config.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_account_edi_ubl
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_account_edi_ubl/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_config
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_config/

dhongu committed:
Update stock report tests and report methods

The update removes an unnecessary import in the test scripts and modifies methods to handle changes in the stock report. A key change is the addition of serial number handling in various methods of the stock report, improving the tracking and tracing of stock movements. It also makes the location field in the stock report no longer mandatory and handles cases where no specific location is given.

OCA-git-bot committed:
Merge PR #758 into 16.0

Signed-off-by dhongu

Commit:
[UPD] Update l10n_ro_stock_report.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_stock_report
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_stock_report/

mcojocaru committed:
[16.0][FIX] actiune inchidere perioada cu target = new

OCA-git-bot committed:
Merge PR #759 into 16.0

Signed-off-by feketemihai

OCA-git-bot committed:
[BOT] post-merge updates

stananatanaela committed:
[16.0] modificare _compute_value pt cazurile l10n_ro_stock_account_svl_lot_allocation=False

OCA-git-bot committed:
Merge PR #763 into 16.0

Signed-off-by feketemihai

Commit:
[UPD] Update l10n_ro_stock_account.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_stock_account
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_stock_account/

dhongu committed:
add ; in neutralize.sql

OCA-git-bot committed:
Merge PR #766 into 16.0

Signed-off-by dhongu

OCA-git-bot committed:
[BOT] post-merge updates

feketemihai committed:
[16.0] [IMP] Improve EDI Ubl

OCA-git-bot committed:
Merge PR #768 into 16.0

Signed-off-by feketemihai

Commit:
[UPD] Update l10n_ro_account_edi_ubl.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_account_edi_ubl
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_account_edi_ubl/

stananatanaela committed:
[16] pt cazurile svl_allocation=False adaug la value costurile aditionale

stananatanaela committed:
[16] pt cazurile svl_allocation=False adaug la value costurile aditionale

dhongu committed:
fix conditie generare factura

dhongu committed:
test numeroatare factura

OCA-git-bot committed:
Merge PR #776 into 16.0

Signed-off-by dhongu

OCA-git-bot committed:
[BOT] post-merge updates

mcojocaru committed:
[FIX][16.0] consumption return sa nu creeze si SVL production

OCA-git-bot committed:
Merge PR #784 into 16.0

Signed-off-by feketemihai

OCA-git-bot committed:
[BOT] post-merge updates

feketemihai committed:
Merge branch '16.0' of github.com:OCA/l10n-romania into 16.0-up_stock_account

OCA-git-bot committed:
Merge PR #775 into 16.0

Signed-off-by feketemihai

OCA-git-bot committed:
[BOT] post-merge updates

adrian-dks committed:
fix error namespace / fix TVA 0

dhongu committed:
precommit

dhongu committed:
_l10n_ro_prepare_invoice_for_download

adrian-dks committed:
[IMP] download PDF

adrian-dks committed:
Fix company

adrian-dks committed:
Merge branch '16.0.fix-anaf-read-invoice' of github.com:adrian-dks/l10n-romania into 16.0.fix-anaf-read-invoice

adrian-dks committed:
Pre-commit

OCA-git-bot committed:
Merge PR #792 into 16.0

Signed-off-by dhongu

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_account_edi_ubl
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_account_edi_ubl/

OCA-git-bot committed:
Merge PR #794 into 16.0

Signed-off-by dhongu

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #790 into 16.0

Signed-off-by dhongu

OCA-git-bot committed:
[BOT] post-merge updates

dhongu committed:
generare edi doc doar la factura de iesire, modificari din v15

OCA-git-bot committed:
Merge PR #795 into 16.0

Signed-off-by dhongu

Commit:
[UPD] Update l10n_ro_account_edi_ubl.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_account_edi_ubl
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_account_edi_ubl/

VictorDakai committed:
Residual Days Improve

VictorDakai committed:
Tratare UserError

VictorDakai committed:
Merge branch '16.0' into 16.0.l10n_ro_account_edi_ubl.residual_days_improve

VictorDakai committed:
Merge branch '16.0' into 16.0.l10n_ro_account_edi_ubl.anaf_userError_on_edi_document

OCA-git-bot committed:
Merge PR #798 into 16.0

Signed-off-by dhongu

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #799 into 16.0

Signed-off-by dhongu

Commit:
[UPD] Update l10n_ro_account_edi_ubl.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_account_edi_ubl
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_account_edi_ubl/

vasi26ro committed:
[MIG] l10n_ro_stock_account_reception_in_progress: Migration to 16.0

dhongu committed:
stergere factura veche

dhongu committed:
pregatire eliminare XML vechi

OCA-git-bot committed:
Merge PR #809 into 16.0

Signed-off-by dhongu

Commit:
[UPD] Update l10n_ro_account_edi_ubl.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_account_edi_ubl
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_account_edi_ubl/

adrian-dks committed:
metoda noua citire lista mesaje anaf

adrian-dks committed:
metoda noua citire lista mesaje anaf

Commit:
[IMP] l10n_ro_stock_account_reception_in_progress: refactor unit test

dhongu commented on l10n_ro_account_edi_ubl/models/res_company.py:
```diff
@@ -61,24 +71,56 @@ def _l10n_ro_get_anaf_efactura_messages(self):
+                zile=zile,
+                start=start,
+                end=end,
+                pagina=doc.get("index_pagina_curenta") + 1,
```
e un bug la anaf si nu te poti baza pe index_pagina_curenta.

OCA-git-bot committed:
Merge PR #807 into 16.0

Signed-off-by dhongu

Commit:
[UPD] Update l10n_ro_stock_account_reception_in_progress.pot

OCA-git-bot committed:
[BOT] post-merge updates

dhongu committed:
fix

dhongu committed:
fix

stananatanaela committed:
[16.0] e-factura: adaugare AE pentru taxarea inversa pe vanzare

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: l10n-romania-16.0/l10n-romania-16.0-l10n_ro_stock_account_reception_in_progress
Translate-URL: https://translation.odoo-community.org/projects/l10n-romania-16-0/l10n-romania-16-0-l10n_ro_stock_account_reception_in_progress/

dhongu committed:
dezactivate temp a testelor

OCA-git-bot committed:
Merge PR #819 into 16.0

Signed-off-by dhongu

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #817 into 16.0

Signed-off-by dhongu

OCA-git-bot committed:
[BOT] post-merge updates

dhongu committed:
Merge remote-tracking branch 'adrian-dks/16.0.fix_read_anaf_list_messages' into 16.0.fix_read_anaf_list_messages
